docs/landing/config.toml

```diff
@@ -2,5 +2,7 @@ baseurl = "/mongo-java-driver/"
 languageCode = "en-us"
 title = "MongoDB Java Driver"
 canonifyurls = false
+disableHugoGeneratorInject = true
+disableKinds = ["section", "taxonomy", "taxonomyTerm", "404"]
```

Comment:
Hugo 0.18 broke `directory/index.md`. So turning off section and taxonomy type pages fixed the issue. I also disabled the 404 page as it isn't used. Only the top level 404 page is used and there isnt a `mongodb/mongodb` repo with gh-pages so there isnt a custom 404 page.